components/ingest-service/habitat/plan.sh

```diff
@@ -10,10 +10,15 @@ pkg_binds=(
   [authz-service]="port"
   [event-service]="port"
   [nodemanager-service]="port"
+  # TODO(ssd) 2019-05-02: We could consider making these optional
```

Comment:
Why make them optional? We already rely on the platform config for the pg URL so if we don't have the binds then ~scheduler~ ~manager~ pg backend isn't ever going to start up.

Reply:
I _think_ my thought when writing this comment was that it seems kinda bad for ingest to fail just because we can't run scheduled tasks.  Ingest is one of the "try to keep it working under all circumstances" services since it is what takes in data.  Ideally, imho, whole feature might live somewhere else.